config-templates/ibm/ibm-vpc-block-csi-driver/changelog.json

```diff
@@ -3,22 +3,24 @@
         "5.0": {
             "status": "supported",
             "latest-revision": {
+                "revision": "11",
+                "release-date": "19 Jun 2023",
+                "cves": "CVE-2022-43552,CVE-2022-3204,CVE-2023-27535,CVE-2022-36227,CVE-2022-35252,CVE-2022-43552,CVE-2022-3204,CVE-2023-27535,CVE-2022-36227,CVE-2022-35252",
```

Comment:
These CVEs are repeated here, please remove the duplicate ones
`CVE-2022-43552,CVE-2022-3204,CVE-2023-27535,CVE-2022-36227,CVE-2022-35252`

Reply:
Add GoLang CVEs 
Ref: https://github.ibm.com/alchemy-containers/armada-storage/issues/4576

Reply:
```CVE-2023-24540
CVE-2023-24539
CVE-2023-29400
CVE-2023-29403
CVE-2023-29404
CVE-2023-29405
CVE-2023-29402```